### PR TITLE
Add bitmasks.1.2.0

### PR DESCRIPTION
**Pull request by dra27**

### CODE REVIEW

File: packages/bitmasks/bitmasks.1.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+license: "BSD-3-clause with OCaml linking exception"
+homepage: "https://metastack.github.io/bitmasks"
+dev-repo: "git+https://github.com/metastack/bitmasks.git"
+bug-reports: "https://github.com/metastack/bitmasks/issues"
+build: [
+  [ "dune" "build" "-p" name "-j" jobs "@install" "@runtest" {with-test}]
+  [ make "doc" ] {with-doc}
+]
+depends: [
+  "ocaml" {< "4.11.0"}
```
Comment (kit-ty-kate) [inline packages/bitmasks/bitmasks.1.2.0/opam:13]: I'm a bit surprised to see this restriction. Previous versions worked fine on OCaml 4.11 and the restriction isn't in the source repository either. Could it be removed?

Comment (dra27) [reply packages/bitmasks/bitmasks.1.2.0/opam:13]: It’s the different between “builds with OCaml 4.11” and “works with 4.11” - you can build it, but you can’t do anything with it. In fact, I should probably push a constraint for 1.1.1 to < 4.07.0 for the same reason!

(it has to match all the functions of Set.S)

Comment (dra27) [reply packages/bitmasks/bitmasks.1.2.0/opam:13]: (https://github.com/metastack/bitmasks/pull/4 is ready to lift the restriction)

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**